extensions/amp-ad/0.1/amp-ad-api-handler.js

```diff
@@ -191,15 +230,9 @@ export class AmpAdApiHandler {
   }
 
   /** @override  */
```

Comment:
Not sure why they were present in the first place, perhaps it's from before amp-ad was refactored to an extension? I'm happy to remove them, but perhaps in a separate change, since it's not logically related to what I'm doing here?